dataland-dummy-qa-service/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+// dataland-dummy-qa-service
+
+val sonarSources by extra(sourceSets.asMap.values.flatMap { sourceSet -> sourceSet.allSource })
+val jacocoSources by extra(sonarSources)
+val jacocoClasses by extra(
+    sourceSets.asMap.values.flatMap { sourceSet ->
+        sourceSet.output.classesDirs.flatMap {
+            fileTree(it) {
+                exclude("**/openApiClient/**")
+            }.files
+        }
+    },
+)
+val jacocoVersion: String by project
+val openApiGeneratorTimeOutThresholdInSeconds: String by project
+
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.spring")
+    jacoco
+    id("com.gorylenko.gradle-git-properties")
+    id("org.springframework.boot")
+    kotlin("kapt")
+    id("org.jetbrains.kotlin.plugin.jpa")
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("org.springframework.boot:spring-boot-starter-amqp")
+    implementation("org.testng:testng:7.7.0")
+    implementation("junit:junit:4.13.1")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.springframework.amqp:spring-rabbit-test")
+    implementation(libs.log4j)
+    implementation(libs.log4j.api)
+    implementation(libs.log4j.to.slf4j)
+    implementation(project(":dataland-message-queue-utils"))
+    implementation(project(":dataland-backend-utils"))
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+}
+
+tasks.test {
+    useJUnitPlatform()
+
+    extensions.configure(JacocoTaskExtension::class) {
+        setDestinationFile(file("$buildDir/jacoco/jacoco.exec"))
+    }
+}
+
+jacoco {
+    toolVersion = jacocoVersion
+    applyTo(tasks.bootRun.get())
```

Comment:
this "applyTo" isn't needed anymore - see e.g. the backend for details (we now use the jacoco agent directly).

Reply:
And it was no more.

Reply:
ok